qemu/tests/cfg/numa_opts.cfg

```diff
@@ -72,15 +75,19 @@
         - defaults:
           extra_params += " -numa node -numa node -numa node"
           # nodes are 8MB-aligned:
-          numa_node0_cpus = "0 3"
+          numa_node0_cpus = "0 1 2"
+          numa_node1_cpus = "3 4 5"
+          numa_node2_cpus = ""
+          Host_RHEL.m6, Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2:
+              numa_node0_cpus = "0 3"
+              numa_node1_cpus = "1 4"
+              numa_node2_cpus = "2 5"
```

Comment:
With 7.2 os and 7.2 qemu.  I still can't run pass the case.
15:52:32 ERROR| TestFail: Wrong CPU set on numa node 0: set([0, 1, 2]). Expected: set([0, 2, 4])
15:52:32 ERROR|
15:52:32 ERROR| FAIL 3-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.numa_opts.nodes.2.defaults -> TestFail: Wrong CPU set on numa node 0: set([0, 1, 2]). Expected: set([0, 2, 4])

15:52:47 ERROR| FAIL 6-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.numa_opts.nodes.3.defaults -> TestFail: Wrong CPU set on numa node 0: set([0, 1, 2]). Expected: set([0, 3])